app/window.go

```diff
@@ -596,6 +596,13 @@ func Title(t string) Option {
 	}
 }
 
+// Centered set the centered position of the VM
+func Centered(c bool) Option {
```

Comment:
I think it could be `var Centered` ? 

Similar of `var Windowed` or `var AnyOrientation`:

https://github.com/gioui/gio/blob/d5f8c860af591025fc1e27174a2e31400492cf1b/app/window.go#L558-L590

Reply:
What about top-left or just left? I suggest adding a Direction type like layout.Direction and add the option

// Gravity controls the direction a window is placed towards.
func Gravity(dir layout.Direction)

Note in the comments that only Windows is supported.